src/ICS26Router.sol

```diff
@@ -69,9 +69,8 @@ contract ICS26Router is IICS26Router, IBCStore, Ownable, IICS26RouterErrors, Ree
         string memory counterpartyId = ics02Client.getCounterparty(msg_.sourceChannel).clientId;
 
         // TODO: validate all identifiers
-        uint64 nanoTimestamp = uint64(block.timestamp * 1_000_000_000);
-        if (msg_.timeoutTimestamp <= nanoTimestamp) {
-            revert IBCInvalidTimeoutTimestamp(msg_.timeoutTimestamp, nanoTimestamp);
+        if (msg_.timeoutTimestamp <= block.timestamp) {
```

Comment:
The implications for sending packets like this would only be that the light client needs to use seconds (since that is what it will check against for a timeout). That seems both reasonable and easy to reason about.

Reply:
sp1 light client is already using seconds for timestamps